src/TableRow.jsx

```diff
@@ -0,0 +1,44 @@
+import moment from "moment";
+import React, { useState } from "react";
+import CustomerProfile from "./CustomerProfile";
+
+
+const TableRow=(props)=>{
+    const [selectedRow, setSelectedRow] = useState(null);
+    const [showProfile, setShowProfile] = useState(false);
+    const [customClass, setCustomClass] = useState("");
+
+    let checkIn = moment(props.value.checkInDate, "YYYY-MM-DD")
+    let checkOut = moment(props.value.checkOutDate , "YYYY-MM-DD")
+    let rowClass = selectedRow === props.value.id ? customClass : "";          
```

Comment:
Looks like we could use `const` for all of these, its best to do that if we know its not going to be reassigned 😄